Droidcon-Boston/app/src/main/AndroidManifest.xml

```diff
@@ -11,16 +11,17 @@
         android:label="@string/app_name"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity
-            android:name=".views.SplashActivity"
+        <activity android:name=".views.SplashActivity"
             android:screenOrientation="sensorPortrait">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
 
-        <activity android:name=".views.MainActivity" />
+        <activity
+            android:name=".views.MainActivity"
+            android:screenOrientation="sensorPortrait" />
```

Comment:
we should probably remove this at some point (currently agenda UI doesn't handle portrait very well)

Reply:
Valid point. We should have a different layout for landscape version if we are going to allow that.

Reply:
Portrait only works!